spec.html

```diff
@@ -4831,7 +4831,9 @@ <h1>ToInteger ( _argument_ )</h1>
         1. Let _number_ be ? ToNumber(_argument_).
         1. If _number_ is *NaN*, *+0*, or *-0*, return *+0*.
         1. If _number_ is *+&infin;*, or *-&infin;*, return _number_.
-        1. Return the Number value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
+        1. Let _integer_ be floor(abs(_number_)).
+        1. If ! SameValueZero(_integer_, *+0*), return *+0*.
```

Comment:
```suggestion
        1. If ! SameValueZero(_integer_, *+0*) is *true*, return *+0*.
```

Reply:
It's kind of weird to use `SameValueZero` here anyway. I would just say `If _integer_ is *-0*, return *+0*`.